### ISSUE TITLE
Fix client debug print statement [cute task]

### ISSUE THREAD

**Issue by evanw555** [Brethren-Studios/chimsockets#101]
**WARNING:** this is a _cute_ task

In `main.js`, the socket handler for `input-complete` has a debug print statement that is incorrect. It currently is:

```
console.log(`*input-complete {status=${data.status}, type={data.type}}`);
```

It should be changed to:

```
console.log(`*input-complete {status=${data.status}, type=${data.type}}`);
```

**Comment by davidvorona** [comment]
For noobs only.

Just kidding, all noobs are welcome.

**Comment by evanw555** [comment]
@davidvorona 

*\*tsyaahahaaaaa\** all noobs are welcome

**Comment by evanw555** [comment]
@davidvorona 

- [ ] Beast mode off
- [x] Beast mode on

```javascript
socket.on('barberituum', (data) => {
    console.log('this js markdown is dope');
});
```

**Comment by davidvorona** [comment]
Issue closed by PR #109.
